docs/index.md

```diff
@@ -0,0 +1,24 @@
+# Welcome to the Guidelines to Join FIWARE Lab
+
+This documentation describes the process to set-up a FIWARE Lab Node
+and to connect it to the FIWARE Lab.
```

Comment:
"...and connect it..." would read better

Reply:
👍